prefect_twitter/tweets.py

```diff
@@ -1,6 +1,91 @@
 """This is a module for interacting with Twitter tweets"""
-from prefect import task
+
+from functools import partial
+from typing import TYPE_CHECKING, List, Optional, Union
+
+from anyio import to_thread
+from prefect import get_run_logger, task
+
+if TYPE_CHECKING:
+    from prefect_twitter import TwitterCredentials
+
 
 @task
-def create_tweet():
-    pass
+async def update_status(
+    twitter_credentials: "TwitterCredentials",
+    status: Optional[str] = None,
+    media_ids: Optional[List[Union[int, str]]] = None,
+    **kwargs: dict
+) -> int:
+    """
+    Updates the authenticating user's current status, also known as Tweeting.
+
+    Args:
+        twitter_credentials: Credentials to use for authentication with Tweepy.
+        status: Text of the Tweet being created. This field is required
+            if media_ids is not present.
+        media_ids: A list of Media IDs being attached to the Tweet.
+        kwargs: Additional keyword arguments to pass to update_status.
```

Comment:
Can you link to some documentation so that users know what other kwargs can be passed to `update_status`?